gateway/api_loader.go

```diff
@@ -751,7 +752,17 @@ func explicitRouteSubpaths(prefix string, handler http.Handler, enabled bool) ht
 //
 // - register gorilla/mux routing handless with proxyMux directly (wrapped),
 // - return a raw http.Handler for tyk://ID urls.
-func (gw *Gateway) loadHTTPService(spec *APISpec, apisByListen map[string]int, gs *generalStores, muxer *proxyMux) *ChainObject {
+func (gw *Gateway) loadHTTPService(spec *APISpec, apisByListen map[string]int, gs *generalStores, muxer *proxyMux) (*ChainObject, error) {
+	var (
+		listenPath = spec.Proxy.ListenPath
```

Comment:
any reason for not using the short asignment operator here ? 1 single line instead of 3 and probably easier to read

Reply:
I think I'll remove it. For functions I like to declare meaningful vars at the start with a var() block in general as it makes the theme more apparent.